src/include/processor/operator/persistent/index_builder_error_handler.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "common/exception/copy.h"
+#include "common/types/types.h"
+#include "processor/execution_context.h"
+
+namespace kuzu {
+namespace storage {
+class NodeTable;
+}
+
+namespace processor {
+template<typename T>
+struct IndexBuilderError {
+    std::string message;
+    T key;
+    common::nodeID_t nodeID;
+};
+
+class IndexBuilderErrorHandler {
```

Comment:
Can u rename this to `NodeBatchInsertErrorHandler`?

Reply:
updated